### PR TITLE
feat: remove bonita 7.10

### PR DESCRIPTION
**Pull request by educhastenier**
Depends on https://github.com/bonitasoft/bonita-doc/pull/2651

### CODE REVIEW

**Comment by educhastenier** [comment]
@tbouffard [this page](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/documentation-components-and-versions.adoc) states to "Remove the version from the DocSearch configuration"
How should I do that?

File: scripts/propagate_doc_upwards.bash [hunk 1]
```diff
@@ -48,7 +48,6 @@ log "Configuration: REPO_NAME=${REPO_NAME}"
 git config merge.ours.driver true
 
 if [ "${REPO_NAME}" == "bonita-doc" ]; then
-  merge "2022.1" "2022.2"
```
Comment (tbouffard) [inline scripts/propagate_doc_upwards.bash:51]: **nitpick:** out of the scope of the 7.10 removal, must be done in a dedicated PR

**Comment by tbouffard** [comment]
> @tbouffard [this page](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/documentation-components-and-versions.adoc?rgh-link-date=2024-04-02T15%3A51%3A57Z) states to "Remove the version from the DocSearch configuration" How should I do that?



> @tbouffard [this page](https://github.com/bonitasoft/bonita-documentation-site/blob/master/docs/documentation-components-and-versions.adoc?rgh-link-date=2024-04-02T15%3A51%3A57Z) states to "Remove the version from the DocSearch configuration" How should I do that?

This documentation is incomplete, the internal documentation must be follow instead.
Let's talk about it in person.

**Review by tbouffard — APPROVED** [review]
